packages/backend/src/server/api/endpoints/meta.ts

```diff
@@ -259,13 +259,16 @@ export default class extends Endpoint<typeof meta, typeof paramDef> {
 		super(meta, paramDef, async (ps, me) => {
 			const instance = await this.metaService.fetch(true);
 
-			const ads = await this.adsRepository.find({
-				where: {
-					expiresAt: MoreThan(new Date()),
-					startsAt: LessThanOrEqual(new Date()),
-				},
-			});
-
+			const ads = await this.adsRepository.createQueryBuilder("ads")
+				.where('ads.expiresAt > :now', { now: new Date() })
+				.andWhere('ads.startsAt <= :now', { now: new Date() })
+				.andWhere(new Brackets(qb => {
+					// 曜日のビットフラグを確認する
+					qb.where('ads.dayOfWeek & :dayOfWeek > 0', { dayOfWeek: Math.pow(2, new Date().getDay()) })
```

Comment:
```suggestion
					qb.where('ads.dayOfWeek & :dayOfWeek > 0', { dayOfWeek: 1 << new Date().getDay() })
```

どちらかというとこの揺れが混乱を助長している気がする

Reply:
というか `new Date` まとめたさある

Reply:
あーたしかに変数にしようとすると逆に見づらくなる感ありそう